src/Controls/tests/TestCases.HostApp/Issues/XFIssue/Issue4714.cs

```diff
@@ -3,7 +3,7 @@ namespace Maui.Controls.Sample.Issues;
 [Issue(IssueTracker.Github, 4714, "SingleTapGesture called once on DoubleTap", PlatformAffected.UWP)]
 public class Issue4714 : TestContentPage
 {
-	const string InitialText = "Click Me To Increment";
+	const string InitialText = "ClickMeToIncrement";
```

Comment:
The test fails using spaces?

Reply:
Mnnn, thanks for the feedback, is fine using AutomationId, will take a look to find it using the text.